README.md

```diff
@@ -84,6 +84,22 @@ or [Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-watson).
 ## Examples
 The [examples][examples] folder has basic and advanced examples.
 
+### Document Conversion integration with Retrieve and Rank
```

Comment:
The examples listed in the readme.md are basic examples. for an advance examples add the instructions in the example file in the `/examples` folder